dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/test/java/org/apache/dubbo/samples/configcenter/DemoServiceIT.java

```diff
@@ -19,19 +19,19 @@
 
 import org.apache.dubbo.samples.configcenter.api.DemoService;
 
+import org.apache.dubbo.config.annotation.Reference;
+import org.apache.dubbo.config.spring.context.annotation.EnableDubbo;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = "classpath:spring/configcenter-consumer.xml")
+@EnableDubbo
```

Comment:
was it necessary to change this file?

Reply:
These changing is for regression test. if not changing,  a ClassCastException will not be trigged running with dubbo 2.7.12 or prior version.